AI/BattleAI/common.h

```diff
@@ -11,8 +11,7 @@
 
 class CBattleCallback;
 
-template<typename Key, typename Val, typename Val2>
-const Val getValOr(const std::map<Key, Val> &Map, const Key &key, const Val2 defaultValue)
+template<typename Key, typename Val, typename Val2> const Val getValOr(const std::map<Key, Val> & Map, const Key & key, const Val2 defaultValue)
```

Comment:
Depend on what other think we might not change it at all or force new line instead.
Problem is that half of our code use new line and other half is doesn't.

This is Uncrustify option:
```nl_template_class               = remove   # ignore/add/remove/force```